### PR TITLE
Feature/api 5631 limit webhook registration

### PR DESCRIPTION
**Pull request by MaciejSzaflik**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-5631-limit-webhook-registration)
- [Jira](https://livechatinc.atlassian.net/browse/API-5631)

## 📓 Description

Added information about limit in webhooks registration. In 3.3, 3.4 in main webhooks section. For 3.2 I added info about limitation at Register Webhook method. For bots I just added note in webhooks actions.

### Content

- New content
- Proofreading/content fixes

### CODE REVIEW

File: content/management/configuration-api/index.mdx [hunk 1]
```diff
@@ -2413,7 +2414,7 @@ Here's what you need to know about **webhooks**:
 
 ### Register Webhook
 
-Registers a webhook for the Client ID (application) provided in the request. Later on, registered webhooks can be [enabled](#enable-license-webhooks) for a given license.
+Registers a webhook for the Client ID (application) provided in the request. Later on, registered webhooks can be [enabled](#enable-license-webhooks) for a given license. Number of webhooks that you can register is limited to three for every action.
```
Comment (Adimus11) [inline content/management/configuration-api/index.mdx:2417]: Shouldn't we introduce this description in `3.3` and higher?

File: content/management/configuration-api/v3.2/index.mdx [hunk 2]
```diff
@@ -1932,6 +1932,10 @@ Here's what you need to know about **webhooks**:
 
 ### Register Webhook
 
+Number of webhooks that you can register is limited and depends on authorization method:
+- while using PAT (using diffrent PATs *doesn't* rise this limit): 10 for every action
```
Comment (Adimus11) [inline content/management/configuration-api/v3.2/index.mdx:1936]: Ar you shure that we distinguish `pats`? AFAIR we just use `pat` in db entries

Comment (MaciejSzaflik) [reply content/management/configuration-api/v3.2/index.mdx:1936]: by "using diffrent PATs *doesn't* rise this limit" I meant that we are not distinguishing PATs -> clientId is "personal_access_token" when someone is using PAT

File: content/management/configuration-api/index.mdx (hunk 1 above)
Comment (MaciejSzaflik) [reply content/management/configuration-api/index.mdx:2417]: I have added it in 3.4 [link]( https://developers.labs.livechat.com/docs/feature/API-5631-limit-webhook-registration/management/configuration-api/v3.4/#webhooks)
and 3.3 [link](https://developers.labs.livechat.com/docs/feature/API-5631-limit-webhook-registration/management/configuration-api/#webhooks)
do you mean some other place?

File: content/management/configuration-api/v3.2/index.mdx (hunk 2 above)
Comment (Adimus11) [reply content/management/configuration-api/v3.2/index.mdx:1936]: Ok, I read it wrong way xD

File: content/management/configuration-api/index.mdx (hunk 1 above)
Comment (Adimus11) [reply content/management/configuration-api/index.mdx:2417]: And also here I forgot that we switched to 3.3 as default

**Review by Adimus11 — APPROVED** [review]

**Pull request merged**